src/bus/mod.rs

```diff
@@ -60,13 +60,20 @@ impl Bus {
                 guid.into()
             }
         };
+
```

Comment:
I think you probably ended up squashing the fixup for the first commit into the second. :)

Reply:
Oh, weird

Fixed!